src/Routes.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Dashboard from './pages/Dashboard';
+import ContactUs from './pages/ContactUs';
+import AppDemo from './pages/AppDemo';
+import Offers from './pages/Offers';
+
+const AppRoutes = () => {
+    return (
+        <Routes>
+            <Route path="/dashboard" element={<Dashboard />} />
+            <Route path="/contact" element={<ContactUs />} />
+            <Route path="/offers" element={<Offers />} />
+            <Route path="/" element={<Dashboard />} />  
+            <Route path="/demo" element={<AppDemo />} />
+            <Route path="/offers/create" element={<Offers />} />
```

Comment:
we should not use the same offers component for edit/add, we should use AddorEditComponent directly here